services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.dateparse import parse_date
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
```

Comment:
The type hint `id` is not valid. Use `int` for `movie_id` to specify that it expects an integer ID.